app/templates/gentelella/users/events/_table.html

```diff
@@ -53,15 +53,15 @@
 
                     </td>
                     <td>
-                        <ol style="padding-left:0;">
+                        <ol style="padding-left:10px;">
                             <li>{{ _("Drafts") }}: {{ drafts.count }}</li>
                             <li>{{ _("Submitted") }}: {{ event.session | length }}</li>
                             <li>{{ _("Accepted") }}: {{ accepted.count }}</li>
                             <li>{{ _("Pending") }}: {{ pending.count }}</li>
                             <li>{{ _("Rejected") }}: {{ rejected.count }}</li>
                         </ol>
                     </td>
-                    <td>{{ event.speaker | length }}</td>
+                    <td style="padding-left:30px;">{{ event.speaker | length }}</td>
```

Comment:
@niranjan94 without that padding it looked like this thats why I did 
![screenshot from 2017-02-10 18 32 54](https://cloud.githubusercontent.com/assets/17119462/22827767/7c892a9c-efbf-11e6-90dc-fa88195f83b6.png)